src/module/config.js

```diff
@@ -1400,6 +1400,51 @@ SFRPG.CR_EXP_LEVELS = [
     1638400
 ];
 
+SFRPG.DifficultyLevels = {
+    "no-pcs": "SFRPG.Combat.Difficulty.Levels.NoPCs",
+    "no-enemies": "SFRPG.Combat.Difficulty.Levels.NoEnemies",
+    "less-than-easy": "SFRPG.Combat.Difficulty.Levels.LessThanEasy",
+    "easy": "SFRPG.Combat.Difficulty.Levels.Easy",
+    "average": "SFRPG.Combat.Difficulty.Levels.Average",
+    "challenging": "SFRPG.Combat.Difficulty.Levels.Challenging",
+    "hard": "SFRPG.Combat.Difficulty.Levels.Hard",
+    "epic": "SFRPG.Combat.Difficulty.Levels.Epic",
+    "greater-than-epic": "SFRPG.Combat.Difficulty.Levels.GreaterThanEpic"
+};
+
+SFRPG.CRMap = new Map();
+SFRPG.CRMap.set("0", {minXP: 0, totalXP: 0, perPlayerXP123: 0, perPlayerXP45: 0, perPlayerXP6: 0});
```

Comment:
How does that work with fractions? If I set a key as 1/4 I assume it'll convert to 0.25, and I'll have to convert that back to a string for display later?

Reply:
That's true. Realistically I just want to avoid using `eval`, perhaps you could parse `"1/2"` into`0.5` by splitting the fraction by the `/`

Reply:
If you don't wanna do numbers, then there's no real reason to use the Map, you can lookup objects just fine with `CRMap["1/2"]`; up to you

Reply:
Okay yeah I just swapped it over to using an object then, I think it'll be a bit simpler this way rather than doing double conversions for the fractional values.